railties/test/application/test_runner_test.rb

```diff
@@ -536,6 +536,21 @@ def test_rake_passes_multiple_TESTOPTS_to_minitest
       assert_match "seed=1234", output, "passing TEST= should run selected test"
     end
 
+    def test_rake_runs_multiple_test_tasks
+      create_test_file :models, "account"
+      create_test_file :controllers, "accountcontroller"
+      output = Dir.chdir(app_path) { `bin/rake test:models test:controllers TESTOPTS='-v'` }
+      assert_match "AccountTest#test_truth", output, "passing test:models should run test/models/"
+      assert_match "AccountcontrollerTest#test_truth", output, "passing test:controllers should run test/controllers"
+    end
+
+    def test_rake_db_and_test_tasks_parses_args_correctly
+      create_test_file :models, "account"
+      output = Dir.chdir(app_path) { `bin/rake db:migrate test:models TESTOPTS='-v' && echo ".tables" | rails dbconsole` }
```

Comment:
The test doesn't hang if that's what you mean, but it may take a bit longer to run. I wanted to check that the db:migrate task ran too, and this is the method used by previous tests.